ci/gpu/build.sh

```diff
@@ -284,6 +284,9 @@ else
     unset LIBCUML_BUILD_DIR
     $WORKSPACE/build.sh cppdocs -v
 
+    gpuci_logger "Building python docs"
```

Comment:
```suggestion
    if [ "$CUDA_REL" == "11.0" ];then
        gpuci_logger "Building python docs"
```

Finally figured out why the CUDA 11.0 job is not woring: that job does not build the python artifacts, so the way to overcome it is to skip this test in the 11.0 job, the 11.2/5 jobs will check it and is sufficient.